### ISSUE TITLE
MIDI-playback support

### ISSUE THREAD

**Issue by greenbech** [naomiaro/waveform-playlist#144]
Thank you for this great project! I frequenctly use the [Working With Local Files](http://naomiaro.github.io/waveform-playlist/newtracks.html) page to analyze and transcribe music created with a music source separation model such as [Spleeter](https://github.com/deezer/spleeter) or [Open-Unmix](https://github.com/sigsep/open-unmix-pytorch) (which works great! 🤘🎶)

Now, I'm working on automatic music transcription and for that usecase it would be very valuable to be able to play along the transcribed MIDI-files with the audio. I'm sure MIDI-playback also would be valuable in many other contexts. 

Any thoughts on adding this?

Here are some other web-based MIDI-sites for inspiration:
- https://github.com/cifkao/html-midi-player
- https://github.com/ryohey/signal

**Comment by naomiaro** [comment]
Wow those Github projects look pretty great!

So you would like the ability to use a midi file as an input source along with an mp3/WAV file?

I could perhaps take a look at a project such as https://github.com/feross/timidity

**Comment by greenbech** [comment]
> So you would like the ability to use a midi file as an input source along with an mp3/WAV file?

Yes, exacty. I would like the tracks in the MIDI file to act just like a waveform track with solo and mute and so on.

> I could perhaps take a look at a project such as https://github.com/feross/timidity

That project definitely looks like a great place to start!

**Comment by isaiahdahl** [comment]
Thumbs up on this one!
